eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/model/VariantNoHgvsLink.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2014-2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.dbsnp.model;
+
+import java.sql.Date;
+
+public class VariantNoHgvsLink {
```

Comment:
I would use a similar nomenclature to the one in the original importer (`SubSnpCoreFields`), making this `SubSnpNoHgvsLinkFields` .

Reply:
left as SubSnpNoHgvs